aicsimageio/writers/timeseries_writer.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from pathlib import Path
+from typing import Union
+
+import dask.array as da
+import numpy as np
+from imageio import get_writer
+
+from ..exceptions import InconsistentShapeError, InvalidDimensionOrderingError
+from ..transforms import reshape_data
+from .writer import Writer
+
+###############################################################################
+
+
+class TimeseriesWriter(Writer):
+    """
+    A writer for timeseries Greyscale, RGB, or RGBA image data.
+    Primarily directed at formats: "gif", "mp4", "mkv", etc.
+    """
+
+    DIM_ORDERS = {
+        3: "TYX",  # Greyscale
+        4: "TYXC",  # RGB
+    }
+
+    @staticmethod
+    def save(
+        data: Union[da.Array, np.ndarray],
+        filepath: Union[str, Path],
+        dim_order: str = None,
```

Comment:
does this dim_order default value override the base class Writer's default value for the same arg?   

I am still slightly of the opinion that we don't really need a base class and all the writers are going to just implement a single save function anyway.  Is there a future world where this gets more complicated and needs classes with instance methods?  At this point the base class is only enforcing that you pass data and filepath in to this static call.

Reply:
It does indeed override the default value. I guess I could move from the base class to simply having single functions.

Reply:
For more complex formats I could see "save subblock metadata" or something though

Reply:
we can discuss.  it's not critical to change it right here in this PR.